docs/rest/openapi_demo.md

```diff
@@ -142,6 +142,75 @@ For example, to create a "sidetree" public DID in alice agent, go to `HTTP POST
+of [how to establish a did-connection](https://github.com/hyperledger/aries-framework-go/blob/master/docs/rest/openapi_demo.md#how-to-create-a-did-connection-through-the-out-of-band-protocol).
+
+1. Send a request presentation (Alice).
+
+    Make sure that did-connection is established and you have the following values:
+    ```
+    MyDID: did:peer:1zQmRmFgG6PUX8d9Zrrkdh9akh3uijnoTMfx8XiRhMDSFJwT
+    TheirDID: did:peer:1zQmQSLFWySB3LACeSrUpvM48QN9frMayNHypnsQjk4GhQKG
+    ```
+    Then perform `/presentproof/send-request-presentation` API call.
+    ```
+    curl -X POST "https://localhost:8082/presentproof/send-request-presentation" -H  "accept: application/json" -H  "Content-Type: application/json" -d "{\"my_did\":\"did:peer:1zQmRmFgG6PUX8d9Zrrkdh9akh3uijnoTMfx8XiRhMDSFJwT\",\"request_presentation\":{},\"their_did\":\"did:peer:1zQmQSLFWySB3LACeSrUpvM48QN9frMayNHypnsQjk4GhQKG\"}"
+    ```
+   The response should be similar to the following:
+   ```json
+   {"piid":"80f8b418-4818-4af6-8915-f299b974f5c2"}
+   ```
+2. Accept a request presentation (Bob).
+
+    To accept a request presentation you need to know `PIID`.
+    You can achieve that by performing `/presentproof/actions` API call.
+    ```
+    curl -X GET "https://localhost:9082/presentproof/actions" -H  "accept: application/json"
+    ```
+    The response should be similar to the following:
+    ```json
+    {"actions":[{"PIID":"80f8b418-4818-4af6-8915-f299b974f5c2","Msg":{"@id":"80f8b418-4818-4af6-8915-f299b974f5c2","@type":"https://didcomm.org/present-proof/2.0/request-presentation","~thread":{"thid":"80f8b418-4818-4af6-8915-f299b974f5c2"}},"MyDID":"did:peer:1zQmQSLFWySB3LACeSrUpvM48QN9frMayNHypnsQjk4GhQKG","TheirDID":"did:peer:1zQmRmFgG6PUX8d9Zrrkdh9akh3uijnoTMfx8XiRhMDSFJwT"}]}
+    ```
+    Then you need to perform `/presentproof/{piid}/accept-request-presentation` API call.
+    ```
+    curl -X POST "https://localhost:9082/presentproof/80f8b418-4818-4af6-8915-f299b974f5c2/accept-request-presentation" -H  "accept: application/json" -H  "Content-Type: application/json" -d "{\"presentation\":{\"presentations~attach\":[{\"lastmod_time\":\"0001-01-01T00:00:00Z\",\"data\":{\"base64\":\"ZXlKaGJHY2lPaUp1YjI1bElpd2lkSGx3SWpvaVNsZFVJbjAuZXlKcGMzTWlPaUprYVdRNlpYaGhiWEJzWlRwbFltWmxZakZtTnpFeVpXSmpObVl4WXpJM05tVXhNbVZqTWpFaUxDSnFkR2tpT2lKMWNtNDZkWFZwWkRvek9UYzRNelEwWmkwNE5UazJMVFJqTTJFdFlUazNPQzA0Wm1OaFltRXpPVEF6WXpVaUxDSjJjQ0k2ZXlKQVkyOXVkR1Y0ZENJNld5Sm9kSFJ3Y3pvdkwzZDNkeTUzTXk1dmNtY3ZNakF4T0M5amNtVmtaVzUwYVdGc2N5OTJNU0lzSW1oMGRIQnpPaTh2ZDNkM0xuY3pMbTl5Wnk4eU1ERTRMMk55WldSbGJuUnBZV3h6TDJWNFlXMXdiR1Z6TDNZeElsMHNJbWh2YkdSbGNpSTZJbVJwWkRwbGVHRnRjR3hsT21WaVptVmlNV1kzTVRKbFltTTJaakZqTWpjMlpURXlaV015TVNJc0ltbGtJam9pZFhKdU9uVjFhV1E2TXprM09ETTBOR1l0T0RVNU5pMDBZek5oTFdFNU56Z3RPR1pqWVdKaE16a3dNMk0xSWl3aWRIbHdaU0k2V3lKV1pYSnBabWxoWW14bFVISmxjMlZ1ZEdGMGFXOXVJaXdpUTNKbFpHVnVkR2xoYkUxaGJtRm5aWEpRY21WelpXNTBZWFJwYjI0aVhTd2lkbVZ5YVdacFlXSnNaVU55WldSbGJuUnBZV3dpT201MWJHeDlmUS4=\"}}]}}"
```

Comment:
It would be nice to show the decoded Presentation Exchange payloads in the text to give the reader more context.